en/mkdocs.yml

```diff
@@ -113,9 +113,13 @@ nav:
     - EMR Connectivity:
       - EMR Systems Overview: emr-connectivity/guides/emr-systems-overview.md
       - Connecting with Epic EMR systems: emr-connectivity/guides/emr-connector.md
-    - Securing a Health APIs:
+    - Securing Health APIs:
       - SMART on FHIR: secure-health-apis/guides/smart-on-fhir.md
-      - Retrive Patient ID: secure-health-apis/guides/getting-patientid.md
+      - Configure SMART App Launch: secure-health-apis/guides/getting-patientid.md
+    - Advanced Topics:
+      - Enable a workflow: advance-topics/guides/enable-workflow.md
```

Comment:
This shouls be "Enable application creation workflow"?

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/36aabfcfb4951987c81145c2be88e9ee45344c9b